installing/installing_azure/preparing-to-install-on-azure.adoc

```diff
@@ -0,0 +1,44 @@
+[id="preparing-to-install-on-azure"]
+= Preparing to install on Azure
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-azure
+
+toc::[]
+
+[id="preparing-to-install-on-azure-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
```

Comment:
Is there a difference between reviewing details about ^^ adnd reading the documentation?

Reply:
Yes, the page that is cross referenced is external to the Installing on Azure subsection and it provides a detailed overview of the installation process. It is foundational reading prior to deciding on which installation method to use.